README.md

```diff
@@ -58,24 +55,81 @@ pip3 install .
 
 ### Configuring the client
 
+#### Define Modules
+Authentication strategies supported by this package are `authn`, `authn-ldap`, `authn-oidc`. Based on authentication machanisam use the below module.
+
+* Authn authentication (Supported on Conjur OSS, Enterprise, Cloud)
+```python
+from conjur_api.providers.authn_authentication_strategy import AuthnAuthenticationStrategy
+```
+
+* OIDC authentication (Supported on Conjur OSS, Enterprise)
```

Comment:
@Nirupma-Verma I thought that we were dropping OIDC support for Python SDK

Reply:
@adamouamani : Support for new version of OIDC v2(OIDC Authenticator for Conjur UI and Conjur CLI authentication) is being dropping, older version v1(OIDC Authenticator for application) with username and id_token is still supported.